Assets/XML/Buildings/CIV4BuildingsSchema.xml

```diff
@@ -121,7 +121,7 @@
 	<ElementType name="iMinAreaSize" content="textOnly" dt:type="int"/>
 	<ElementType name="iConquestProb" content="textOnly" dt:type="int"/>
 	<ElementType name="iCitiesPrereq" content="textOnly" dt:type="int"/>
-	<ElementType name="iColoniesPrereq" content="textOnly" dt:type="int"/> <!-- Leoreth -->

```

Comment:
Please keep my comments here, they are helpful in figuring out possible causes of errors.